proposals/0004-nio-redis.md

```diff
@@ -180,7 +181,7 @@ This `Logger` instance can be overridden in the `init`.
 public final class RedisMessageEncoder: MessageToByteEncoder {
     public typealias OutboundIn = RESPValue
 
-    public init(logger: Logger = Logger(label: "NIORedis.RedisMessageEncoder"))
+    public init(logger: Logger = Logger(label: "RedisNIO.RedisMessageEncoder"))
```

Comment:
Maybe we should standardize on using a URI scheme for this? i.e., `com.mordil.redis-nio...` ?

cc @weissi

Reply:
Perhaps we should have a general recommendation defined - but I think this is more innocuous than say, the package & module names that have hard technical requirements behind them.

It's also different in that throughout all of RedisNIO, where Loggers are created they can always be overridden by the user in case they want to have their own instances w/ metadata they set or to have a label of their choosing. [Even in the `makeConnection` factory method](https://gitlab.com/Mordil/swift-redis-nio-client/blob/master/Sources/RedisNIO/Redis.swift#L82)